editoast/src/views/timetable.rs

```diff
@@ -269,6 +276,44 @@ async fn train_schedule(
     Ok(Json(train_schedule.into_iter().map_into().collect()))
 }
 
+/// Create paced trains by batch
+#[utoipa::path(
+    post, path = "",
+    tag = "timetable,paced_train",
+    params(TimetableIdParam),
+    request_body = Vec<PacedTrainBase>,
+    responses(
+        (status = 200, description = "The created paced trains", body = Vec<PacedTrainResult>)
+    )
+)]
+async fn paced_train(
+    State(db_pool): State<DbConnectionPoolV2>,
+    Extension(auth): AuthenticationExt,
+    Path(TimetableIdParam { id: timetable_id }): Path<TimetableIdParam>,
+    Json(paced_trains): Json<Vec<PacedTrainBase>>,
+) -> Result<Json<Vec<PacedTrainResult>>> {
+    let authorized = auth
+        .check_roles([BuiltinRole::TimetableWrite].into())
+        .await
+        .map_err(AuthorizationError::AuthError)?;
+    if !authorized {
+        return Err(AuthorizationError::Forbidden.into());
+    }
+
+    let conn = &mut db_pool.get().await?;
+
+    Timetable::exists(conn, timetable_id).await?;
```

Comment:
I think exists returns a boolean with the result
```suggestion
    let is_timetable_exists = Timetable::exists(conn, timetable_id).await?;
    if !is_timetable_exists {
        return Err(TimetableError::NotFound { timetable_id: timetable_id }.into())
    }
```